bitbucket/provider_test.go

```diff
@@ -31,9 +31,12 @@ func TestProvider_impl(t *testing.T) {
 
 func testAccPreCheck(t *testing.T) {
 	if v := os.Getenv("BITBUCKET_USERNAME"); v == "" {
-		t.Fatal("BITBUCKET_USERNAME must be set for acceptence tests")
+		t.Fatal("BITBUCKET_USERNAME must be set for acceptance tests")
 	}
 	if v := os.Getenv("BITBUCKET_PASSWORD"); v == "" {
-		t.Fatal("BITBUCKET_PASSWORD must be set for acceptence tests")
+		t.Fatal("BITBUCKET_PASSWORD must be set for acceptance tests")
+	}
+	if v := os.Getenv("BITBUCKET_TEAM"); v == "" {
```

Comment:
Not that my opinion matters at all, but I like the idea of introducing a `BITBUCKET_TEAM` environment variable.